### ISSUE TITLE
AttributeError: 'HierarchyObject' object has no attribute '_entity'

### ISSUE THREAD

**Issue by dattran96** [alexforencich/verilog-uart#9]
It throws me an error while I'm trying to make the testbench:

```
cd verilog-uart/tb/uart_tx
make
```

Error is:
**AttributeError: 'HierarchyObject' object has no attribute '_entity'**

You have any idea how can I fix it? Thank you a lot!

**Comment by alexforencich** [comment]
Yeah, looks like I need to fix a few things in this repo due to changes in some of the simulation components.

**Comment by alexforencich** [comment]
Should be fixed now so it works with the current version of cocotbext-axi.
